src/libraries/System.CodeDom/src/System.CodeDom.csproj

```diff
@@ -7,9 +7,9 @@
   </PropertyGroup>
   <!-- DesignTimeBuild requires all the TargetFramework Derived Properties to not be present in the first property group. -->
   <PropertyGroup>
-    <IsPartialFacadeAssembly Condition="$(TargetFramework.StartsWith('net4'))">true</IsPartialFacadeAssembly>
+    <IsPartialFacadeAssembly Condition="'$(TargetFramework)' == 'net461'">true</IsPartialFacadeAssembly>
   </PropertyGroup>
-  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+  <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true'">
```

Comment:
curious on why choose IsPartialFacadeAssembly over the TargetFramework ?

Reply:
`IsPartialFacadeAssembly` is more expressive in this case as it indicates that the partial facade assembly doesn't contain any source where as the non partial facade assembly only contains source and no type forwards.